packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -133,12 +134,12 @@ export function getMomentTemplateBanner(
                                 css={
                                     isUsEoyGivingTuesBanner
                                         ? [
-                                              styles.desktopVisualContainer,
+                                              styles.desktopVisualContainer(isNewYearBanner),
                                               styles.desktopGivingTuesVisualContainer,
                                           ]
                                         : isUsEoyV3Banner
                                         ? styles.desktopUsEoyV3Container
-                                        : styles.desktopVisualContainer
+                                        : styles.desktopVisualContainer(isNewYearBanner)
```

Comment:
I'm responsible for a fair few of the dynamic styles in this repo! But going forward I think we want to phase these out in favour of composition to make the transition to 'the template to rule them all' a bit easier

```suggestion
                                        : [ styles.desktopVisualContainer,
                                        isNewYearBanner &&  styles.desktopVisualContainerNY 
                                        ]
```

Reply:
Yep, understood remove parameter to use 2 simpler css parts. All done.